File: StudyNotes/Reverse Engineering/iOS Application/isjailbreak/IsJailbreak/ViewController.m
//
//  ViewController.m
//  IsJailbreak
//
//  Created by 朱双泉 on 2019/6/3.
//  Copyright © 2019 Castie!. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];

    if ([[NSFileManager defaultManager] fileExistsAtPath:@"/Applications/Cydia.app"]) {
        NSLog(@"此设备已越狱");
    } else {
        NSLog(@"此设备未越狱");
    }
}


@end

File: StudyNotes/Swift Note/LiveBroadcast/Server/LiveBroadcastServer/ViewController.swift
//
//  ViewController.swift
//  LiveBroadcastServer
//
//  Created by 朱双泉 on 2018/12/13.
//  Copyright © 2018 Castie!. All rights reserved.
//

import Cocoa

class ViewController: NSViewController {

    @IBOutlet weak var hintLabel: NSTextField!
    fileprivate lazy var serverMgr : ServerManager = ServerManager()
    
    override func viewDidLoad() {
        super.viewDidLoad()
    }
    
    @IBAction func startServer(_ sender: NSButton) {
        serverMgr.startRunning()
        hintLabel.stringValue = "服务器已经开启ing"
    }
    
    @IBAction func stopServer(_ sender: NSButton) {
        serverMgr.stopRunning()
        hintLabel.stringValue = "服务器未开启"
    }
}

